### PR TITLE
added tests for my latest PR

### PR DESCRIPTION
**Pull request by ArtskydJ**
Added a few tests for my latest PR.

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -56,3 +56,39 @@ test('friendly names', function(t) {
...
+	t.end()
+})
+
+test('set and delete return', function (t) {
+	var map = new KeyMaster()
+	
+	t.equal(map.set('key1', 3), 3)
+	t.equal(map.set('key2'), undefined)
+	t.ok(map.remove('key1'))
+	t.notOk(map.remove('key1'))
```
Comment (TehShrike) [inline test.js]: Delete always returns true unless the property is configurable.  If we want `remove` to return based on whether or not the property existed, we'll have to change the code.

Comment (ArtskydJ) [inline test.js]: I was mistaken on how `delete` worked. I don't really care what `remove()` returns.

Comment (TehShrike) [inline test.js]: If this test just changed from `t.notOk` to `t.ok` I'd be fine with it, I don't feel like making `map.remove` return anything different at the moment.

**Comment by TehShrike** [comment]
Thanks for the tests! :-)

**Comment by ArtskydJ** [comment]
You're welcome. :smiley:

**Pull request merged**